src/activities/AddGeoreferenceToPdf.ts

```diff
@@ -0,0 +1,175 @@
+import type { IActivityHandler } from "@vertigis/workflow";
+import {
+    PDFArray,
+    PDFDict,
+    PDFDocument,
+    PDFName,
+    PDFNumber,
+    PDFString,
+} from "pdf-lib";
+
+export interface AddGeoreferenceToPdfInputs {
+    /**
+     * @description The source PDF document to add the georeference metadata to. Either a file path or byte content are accepted.
+     * @required
+     */
+    source: Blob;
+
+    /**
+     * @description The bounding coordinates of the map element to georeference in PDF user space.
+     * The array must contain two coordinate pairs in the following order: [[bottom-left x, bottom-left y], [top-right x, top-right y]].
+     * The coordinate [0, 0] represents the bottom left corner of the page.
+     * Typically the units of a PDF are in points (1/72 of an inch).
+     * @required
+     */
+    pageBounds: number[][];
+
+    /**
+     * @description The bounding coordinates of the map element to georeference in geographic space.
+     * The array must contain four coordinate pairs in the following order: [[bottom-left latitude, bottom-left longitude], [top-left latitude, top-left longitude], [top-right latitude, top-right longitude], [bottom-right latitude, bottom-right longitude]].
+     * @required
+     */
+    mapBounds: number[][];
+
+    /**
+     * @description The Well-Known Text (WKT) definition of the coordinate system of the map coordinates.
+     * @required
+     */
+    coordinateSystem: string;
+
+    /**
+     * @description The zero-based index of the page to apply the georeference metadata to. The default is 0.
+     */
+    pageIndex?: number;
+
+    /**
+     * @description An optional name to apply to the georeference metadata. The default is "Map".
+     */
+    name?: string;
+}
+
+export interface AddGeoreferenceToPdfOutputs {
+    /**
+     * @description A new PDF document containing the specified georeference metadata.
+     */
+    result: Blob;
+}
+
+/**
+ * @category PDF
+ * @displayName Add Georeference To PDF
+ * @description Adds georeference metadata to a PDF document.
+ * @clientOnly
+ * @supportedApps EXB, GWV, GVH, WAB
```

Comment:
```suggestion
 * @supportedApps EXB, VSW, GVH, WAB
```
The designer has recognized `VSW` since v5.36.

Reply:
This activity pack claims to support `5.33`. Let's save that for an across the board breaking change. We don't want one activity in this pack to be different, and we don't want a breaking change for an otherwise non-breaking feature.

Reply:
OK, fair enough.